src/libais/ais.h

```diff
@@ -1566,9 +1566,49 @@ class Ais8_200_10 : public Ais8 {
   Ais8_200_10(const char *nmea_payload, const size_t pad);
 };
 
-// 21 and 22 do not exist
+// vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf
+// ETA at lock/bridge/terminal
```

Comment:
Add a comment as to where to look up the UN country, UN location, Fairway, and Terminal codes

I don't see them right off in https://www.ccr-zkr.org/files/documents/ris/vtt12_e.pdf

Reply:
Just founded this: http://www.ris.eu/docs/File/427/implementation_location_code_austria.pdf